infra/azure/outputs.tf

```diff
@@ -16,6 +16,8 @@ output "global_config" {
       test_storage_uri = "hail-az://${azurerm_storage_account.test.name}/${azurerm_storage_container.test.name}"
     }
     azure = {
+      # FIXME Just for testing
+      gsuite_organization = "broadinstitute.org"
```

Comment:
should we eliminate this?

Reply:
yes I can remove it. I think you'll have to address this in the AAD oauth work.